### PR TITLE
[Back 552] Enhance test cases : ETH / ERC20 / XRP Account Sync Histo Balance

### PR DESCRIPTION
**Pull request by RomanWlm**
Bring test coverage from 68.90% to 76.43% 🐙

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -89,6 +89,7 @@ jobs:
             CORE_PG_PWD: ledger_secret
             CORE_PG_DB_NAME_PREFIX: wdtest_
             CORE_PG_CONNECTION_POOL_SIZE_PER_WALLET_POOL: 2
+            SBT_OPTS: -Xmx3200m -Xss2M -XX:MaxMetaspaceSize=1024M
```
Comment (luchenyuxx) [inline .circleci/config.yml:92]: Xss to 2m is too small,
the default is 128m if I recall correctly

File: src/main/scala/co/ledger/wallet/daemon/models/Pool.scala [hunk 2]
```diff
@@ -278,10 +278,13 @@ class Pool(private val coreP: core.WalletPool, val id: Long) extends Logging {
               walletConfig.putString("BLOCKCHAIN_EXPLORER_VERSION", version)
             case _ =>
           }
-          new URL(s"${path.host}:${path.port}").toString
-        case None => ConfigurationDefaults.BLOCKCHAIN_DEFAULT_API_ENDPOINT
+          new URL(s"${path.host}:${path.port}")
+        case None => new URL(s"${ConfigurationDefaults.BLOCKCHAIN_DEFAULT_API_ENDPOINT}:433")
```
Comment (luchenyuxx) [inline src/main/scala/co/ledger/wallet/daemon/models/Pool.scala:282]: should not hard code.

Comment (RomanWlm) [reply src/main/scala/co/ledger/wallet/daemon/models/Pool.scala:282]: Thanks !! Good catch, this has been commited by mistake

File: .circleci/config.yml (hunk 1 above)
Comment (RomanWlm) [reply .circleci/config.yml:92]: Xss is the stack size, I think default is 4M, I'll remove this setting

**Pull request merged**